app/styles/app/modules/dashboard-row.sass

```diff
@@ -55,7 +55,6 @@
     width: 100%
     bottom: 0
     overflow: hidden
-    z-index: 0
```

Comment:
This is a commented-out element for now, the proper `z-index` can be added if/when it’s restored. 😀